recipes/xs/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: xs
```

Comment:
xs is very generic, I'm afraid of namespace clashes, can we name it more verbose?

Reply:
Technically it should be ok. There are no conflicts at present, excepting perl-xs. Two letters is unusually short though.

Reply:
It could also be capitalised to XS, since that's what the developer calls it? But that might be even worse